content/GPR/GPR_physical_properties.rst

```diff
@@ -0,0 +1,98 @@
+===================
+
+In order to better understand the link between the dielectric permittivity (:math:`\varepsilon`) and electromagnetic waves, let us consider a microwave oven.
+We know that microwaves heat food using electromagnetic radiation, but how do they work?
+
+
+.. figure:: images_new/GPR_microwave_1.jpg
+	:align: right
+	:name: asdf
+	:figwidth: 50%
+	
+	Electromagnetic waves in a microwaves.
+
+
+Within a microwave oven, there is a source which generates high-frequency electromagnetic waves (f = 2.45 GHz).
+These waves move at roughly the speed of light (c = :math:`3.00 \times 10^8` m/s) and have wavelengths of about 12.2 cm.
+We can see from the figure on the right, that EM waves support electric and magnetic fields which oscillate.
+
+
+The walls within the microwave are made out of materials which reflect elecromagnetic waves.
+As a result, the walls designed not to heat up while the microwave is on.
+
+
+.. figure:: images_new/GPR_EMwaves.png
+	:align: right
+	:figwidth: 50%
+	
+	The electric and magnetic fields of an electromagnetic wave.
+
+
+All foods contain water to varying degrees.
+Water is unique from most other molecules because it is naturally polarized (carries an electric dipole moment).
+This property of water molecules makes them align with electric fields very easily; giving them a :ref:`high dielectric permittivity <GPR_table_velocity>` (:math:`\varepsilon_r` = 80).
+When electromagnetic waves come into contact with the water molecules in the food, the water molecules physically re-orient with the oscillation of the wave.
+This is because of the electric field which the wave carries.
+For waves at 2.45 GHz, this is occurring over 2 billion times per second!
+
+
+.. figure:: images_new/GPR_water_molecules.png
+		:align: center
+		:figwidth: 80%
+		
+		Polarization of water molecules. (A) Disordered water molecules. B) Water molecules rotate to align with E-field. C) Water molecules polarized under E-field.
+
+
+
+The magic of microwaves is their operating frequency.
+At roughly 2.45 GHz, water molecules absorb the energy from electromagnetic waves and become agitated.
+The heat energy generated by the physical movement of the water molecules is then transferred throughout the food.
+Therefore, microwaves work by heating water molecules with electromagnetic radiation.
+
+What about the turntable or glass of the microwave? 
+The glass and plastic which make up the window and turntable of the microwave have low dielectric permittivities.
+As a result, these materials do not become polarized by the incoming microwaves, making them non-reactive.
+
+So what have we learned from this example?
+
+	- Very conductive materials will reflect incoming electromagnetic waves.
+	- Materials containing water will become polarized from electromagnetic waves.
+	- Some materials do not absorb electromagnetic radiation easily.
+	- The operating frequency of the source has a great impact on the response of materials.
+
+
```

Comment:
I would suggest including the table of values here 

`.. include:: table_velocity.rst`